docs/pages/guides/migrating-to-react-native-firebase.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Migrating from Expo Firebase to React Native Firebase
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+The `expo-firebase-analytics` and `expo-firebase-recaptcha` packages allowed the use of some Firebase features on Classic Builds, Expo Go, and Expo for Web, but developers often encountered version conflicts when trying to use them alongside the rest of the Firebase suite. With EAS Build and Development Builds, it is much easier to use [React Native Firebase](https://rnfirebase.io/) directly in your entire development workflow, so these packages have been deprecated.
+
+In this guide, we will migrate from the Expo Firebase packages to React Native Firebase by:
+- Replacing the Expo Firebase packages with their React Native Firebase equivalents
+- Replacing the use of Expo Go for testing with a development build customized to your project's needs
+- Testing the updated code on the development build
+
+> **Note:** These instructions assume you have already been successfully using the Expo Firebase packages. If you're adding Firebase to your app for the first time, follow our [Using Firebase](/guides/using-firebase) guide.
+
+## Install React Native Firebase
+Run the following commands to remove `expo-firebase-analytics` and install the React Native Firebase packages:
+
+<Terminal cmd={[
+'# Remove Expo Firebase packages',
+'$ yarn remove expo-firebase-analytics',
+'',
+'# Install RN Firebase, etc.',
+'$ yarn add @react-native-firebase/app @react-native-firebase/analytics'
+]} />
+
+If you are using [Expo Prebuild](/guides/adopting-prebuild) (also known as Managed Workflow) and building your app for iOS, install the `expo-build-properties` plugin and apply the following configuration to your [Expo config]() file.
```

Comment:
As I recall the new term for "managed workflow" was "using prebuild", as you're depending on EAS Build to run the prebuild step for you on each build to sync your config. If you have native project files (by "ejecting" or running prebuild one time and committing the files), EAS Build will skip the prebuild step, as it will assume your committed native project files shouldn't be overwritten.

My thinking on calling out "managed workflow" is that folks using this migration guide might still be working their way over to this new world with its new terms, perhaps even not being far removed from using classic builds.

Whatever the right terms are, I think at this point it makes sense to highlight the fork in the road depending on whether you're (what we used to call) "managed" or "bare," for the user who might be using Expo Go with Bare Workflow.